common/namespace/cache.go

```diff
@@ -81,6 +80,20 @@ const (
 )
 
 type (
+	private struct{}
+
+	ClusterInfo interface {
+		IsGlobalNamespaceEnabled() bool
+		GetCurrentClusterName() string
+	}
+
+	Persistence interface {
```

Comment:
Do these interfaces need to be public?

Reply:
They're required by the NewNamespaceCache constructor so I think so